app/styles/base.less

```diff
@@ -347,7 +347,7 @@ thead {
   width: 22px;
   height: 22px;
   display: inline-block;
-  background: url(img/log.png) no-repeat;
+  background: url(../img/log.png) no-repeat;
```

Comment:
On relative urls, have you tried in both `localhost:5001/webui` and `localhost:port/ipfs/Qmhash` stuff and so on? Just to make sure the imports don't get messed